### PR TITLE
base_location_city module

### PR DESCRIPTION
**Pull request by laszlokr**

### CODE REVIEW

File: base_location_city/report/invoice_document.xml
Comment (janikvonrotz) [inline base_location_city/report/invoice_document.xml:1]: What is this?

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: base_location_city/report/invoice_document.xml
Comment (laszlokr) [reply base_location_city/report/invoice_document.xml:1]: @janikvonrotz apologies, this was left from the module scaffolding, it must be removed

**Review by janikvonrotz — CHANGES_REQUESTED** [review]
The update of the `README.md` is missing.

File: base_location_city/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Base Location City",
+    "summary": """
+        Location coodrinates for cities.
```
Comment (janikvonrotz) [inline base_location_city/__manifest__.py:4]: There is a typo.

File: base_location_city/models/res_city.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class ResCity(models.Model):
+    _inherit = "res.city"
+
+    latitude = fields.Float(string="Latitude", digits=(16, 13))
```
Comment (janikvonrotz) [inline base_location_city/models/res_city.py:7]: Why did you add string?

Also where are the translations?
